R/repr_matrix_df.r

```diff
@@ -212,7 +212,7 @@ repr_latex.data.frame <- repr_latex.matrix
 #' @name repr_*.matrix/data.frame
 #' @export
 repr_text.matrix <- function(obj, ...)
-	paste(utils::capture.output(print(ellip.limit.arr(obj))), collapse = '\n')
+	paste(utils::capture.output(print(ellip.limit.arr(as.data.frame(obj)))), collapse = '\n')
```

Comment:
this should maybe be split into multiple lines to be less complex… and it should be explained why dataframe coercion is necessary